pom.xml

```diff
@@ -72,6 +71,15 @@
             <type>pom</type>
             <scope>import</scope>
         </dependency>
+        <dependency>
```

Comment:
why are these being added again? We already have them in the pom

Reply:
I moved them to the bottom to organise the pom. Git just shows it as being removed near the top and added here :)

Reply:
My bad!